src/content/docs/en-US/events/guides/intersecteventsystemtutorial.md

```diff
@@ -0,0 +1,662 @@
+more types of boxes (**Events**): a “Spike Trap” **Event**, a “Button” **Event**, a “Quest” **Event** and
+numerous other things that will make our life easier when making our game. The box is
+completed, but it is still empty! Now we need to look at the “clay” within the box, also known as
+the **Event Commands**.
+
+[Event Commands](../events/eventcommands.md): Whenever we create an **Event** (Box), it is always empty, and we are holding
+a big block of clay. What we need to do is “break and mold the clay” into several objects. We are choosing the shapes and forms of the broken clay by molding it in different ways. After
+we are done, we put it in the box. To clarify what we are talking about in game terms, this is
+choosing Event commands and putting them inside the Event. When the Event activates, it will
+run all the Event Commands inside it. Let us look at an example of placing the clay (Event
+Commands) within the box (Event):
+
+**Event Commands Steps**
+
+1. What do we want the “Person” to say?
+   - We choose the “Show Text” Command to make Lady Marie say “Hello.”
+
+2. Do we want Lady Marie to do anything else?
+   - We choose “Play Sound” to have her play the “Hello.wav”
+
+Now, when the player interacts with Lady Marie’s, she is going to greet up in both text and
+sound. This is just one example of how we can create different boxes and put things into them.
+We created a “Person Box”, decorated it, named it, and put objects in it that a “Person Box”
+could have in it.
+
+As we continue in this tutorial, we are going to go through all the [Event Commands](../events/eventcommands.md) to show what
+we can create with **Event**. Hint, there will be quite a bit. You might feel a bit overwhelmed at
+this point, but do not worry because we will get through it together. This is just to get you
+familiar with how **Event** and [Event Commands](../events/eventcommands.md) work together. When we start going through
+each [Event Commands](../events/eventcommands.md) and the structures of the **Event**, it will be clearer with how it all comes
+together.
+
+---
+
+## Anatomical Structure of an Event
+
+Now that we got all the terminology out of the way, is time for us to learn how to create an
+**Event**. In this section, we will be looking at the overall structure of an **Event**. Then we will also
+be taking a look at each individual [Event Commands](../events/eventcommands.md) to gain a better understanding of how it
+operates on its own and overall within the **Event**. Most of the [Event Commands](../events/eventcommands.md) are
+straightforward; however, there will also be **Events** that are more complex than their initial
+nature suggests. Let us jump right in at looking at how to even create an **Event**:
+
+<p align="center">Open the Intersect Editor
+	
+![Intersect_Editor_JQAk6nnDxq](https://github.com/PyroTech03/Intersect-Documentation/assets/13249558/514e2184-25e6-493a-ad8a-f8d6dd466ea3)
```

Comment:
Can you change the text inside the `[]` to say what the image actually is supposed to be instead of `Intersect_Editor_JQAk6nnDxq`? There is also `msedge` and `firefox` garbage strings. This text is supposed to be the alt text.

Reply:
Alright.  I'll have to go through it and check these.  I was just doing the image upload and assumed it was a github file naming or something.  I'll keep this in mind for the future as well.